ERROR   2022-05-28 16:14:51,928 [main] Main  - error: 
org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.EMPLEADO(ID) ( /* key:1 */ 1, 'federicodiaz@gmail.com', 'Federico', 'Argentina', 'Diaz')"; SQL statement:
INSERT INTO EMPLEADO (ID, EMAIL, NAME, COUNTRY, LASTNAME) VALUES (1,'agustinapodesta@gmail.com','Agustina','Brasil','Podesta') [23505-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:508)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.mvstore.db.MVPrimaryIndex.add(MVPrimaryIndex.java:120)
	at org.h2.mvstore.db.MVTable.addRow(MVTable.java:519)
	at org.h2.command.dml.Insert.insertRows(Insert.java:174)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.dml.DataChangeStatement.update(DataChangeStatement.java:74)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at Main.main(Main.java:50)
DEBUG   2022-05-28 16:43:55,612 [main] Main  - 1federicodiaz@gmail.comFederico
ERROR   2022-05-28 16:47:16,523 [main] Main  - error: 
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database may be already in use: null. Possible solutions: close all other connection(s); use the server mode [90020-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:165)
	at org.h2.mvstore.db.Store.<init>(Store.java:144)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at Main.getConnection(Main.java:33)
	at Main.main(Main.java:44)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /home/adrianmoncada/test.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 11 more
ERROR   2022-05-28 16:47:29,016 [main] Main  - error: 
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database may be already in use: null. Possible solutions: close all other connection(s); use the server mode [90020-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:165)
	at org.h2.mvstore.db.Store.<init>(Store.java:144)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at Main.getConnection(Main.java:33)
	at Main.main(Main.java:44)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /home/adrianmoncada/test.mv.db [2.1.212/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:163)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:449)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4118)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 11 more
DEBUG   2022-05-28 16:47:39,722 [main] Main  - ID: 1 MAIL: federicodiaz@gmail.com NAME: Federico COUNTRY: Argelia LASTNAME: Diaz
DEBUG   2022-05-28 16:54:39,626 [main] Main  - ID: 1 MAIL: federicodiaz@gmail.com NAME: Federico COUNTRY: Argentina LASTNAME: Diaz
ERROR   2022-05-28 16:54:39,643 [main] Main  - error: 
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:3536)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3522)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:319)
	at Main.main(Main.java:76)
DEBUG   2022-05-28 16:56:54,936 [main] Main  - ID: 1 MAIL: federicodiaz@gmail.com NAME: Federico COUNTRY: Argentina LASTNAME: Diaz
ERROR   2022-05-28 16:56:54,951 [main] Main  - error: 
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:3536)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3522)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:319)
	at Main.main(Main.java:76)
DEBUG   2022-05-28 16:58:29,014 [main] Main  - ID: 1 MAIL: federicodiaz@gmail.com NAME: Federico COUNTRY: Argentina LASTNAME: Diaz
